File: PSet1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){
    
    int height;
    int space;
    int sharp;

    do 
    {
        printf("Height:");
        height = get_int();
    }
    while (height <= 0 || height >= 24);
    for (int i = 1; i < height; i++){
        for (space = (height - i); space >= 0; space--){
            printf(" ");
        }
        for (sharp = 1; sharp <= (i + 1); sharp++){
            printf("#");    
        }
        printf("\n");
    }
    return 0;
}

File: PSet6/mario.py
import cs50
import math

while True:
    print('Height: ', end="")
    height = int(input())
    if(height > 0 and height <= 23):
        for i in range(2, height + 1):
            space = height - i
            print(' ' * space, '#' * i)
        print('#' * (height + 1))
        break
    
    else:
        print('Height: ', end="")
        height = int(input())